### PR TITLE
react-intl

### PR DESCRIPTION
**Pull request by langpavel**

### CODE REVIEW

**Comment by langpavel** [comment]
`react-intl` implementation.
 - enabled languages in configuration
 - extract messages from source automation, just run `npm run extractMessages` and all localizable strings apear in `src/messages/` directory. In fact this is not needed. When you running development server, all messages are extracted and files watched
 - unused localization detection (tracking source files)
 - language negotiation:
      1. `?lang=xx` query string parameter
      2. cookie
      3. request `Accept-Language` header
 - redux action to language live switching (works on server; with little effort will be on client too...)
 - `graphql` endpoint for fetching localization messages

File: package.json [hunk 1]
```diff
@@ -16,12 +16,15 @@
...
     "express-jwt": "3.3.0",
+    "express-request-language": "^1.1.4",
     "fastclick": "1.0.6",
-    "fbjs": "0.8.1",
+    "fbjs": "0.8.0",
     "front-matter": "2.0.7",
     "graphiql": "0.7.0",
     "graphql": "0.5.0",
-    "history": "2.1.0",
+    "history": "2.0.2",
```
Comment (koistya) [inline package.json]: It seems like this branch is a little behind the `master`.

Comment (langpavel) [inline package.json]: Oh sorry, my bad.. I'm fixing this now..

Comment (koistya) [inline package.json]: Never mind, I will merge the `master` into it in a moment.

Comment (langpavel) [inline package.json]: Merging master will not help..

**Pull request merged**